src/MICore/Debugger.cs

```diff
@@ -71,6 +89,7 @@ public bool IsClosed
         private LinkedList<string> _initialErrors = new LinkedList<string>();
 
         protected bool _connected;
+        protected int _localDebuggerPid = -1;
```

Comment:
nit: I know we made this mistake already in other places, but personally I don't like protected assignable fields. Suggestion: make this private and add a set method.